charts/operator/values.yaml

```diff
@@ -0,0 +1,26 @@
+image:
+  repository: "us.gcr.io/rookout/rookout-k8s-operator"
+  tag: "1.0"
+  pullPolicy: Always
+
+operator:
+  matchers:
+    - deployment : "example_deployment_matcher"
+      container: "example_container_matcher"
+      labelels:
+        - name: "deployment label name"
+          value: "deployment label value"
```

Comment:
I think that needs to be removed so it wont use it automatically and add it to the readme's table